confidential/data/confidential.json

```diff
@@ -1,5 +1,5 @@
 {
-  "unblind": [
+  "UnblindOutputTC": [
```

Comment:
Use `unblindOutput`. Generally, we use the name of the method to test, but not capitalized.

Reply:
Done